.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDatePeriodParser.cs

```diff
@@ -459,18 +474,6 @@ private DateTimeResolutionResult ParseSimpleCases(string text, DateObject refere
             }
             else
             {
-                match = this.config.SpecialMonthRegex.MatchExact(text, trim: true);
```

Comment:
Where did this go?

Reply:
This pattern is now parsed in ParseOneWordPeriod (in the if clause starting at line 1167: "if (this.config.IsMonthOnly(trimmedText))")